.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js:
+  - "stable"
+cache:
+  directories:
+  - node_modules
+script:
+  - npm test
+  - npm start
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $github_token
+  local_dir: build
+  on:
+    branch: react-redux-homework
+after success:
```

Comment:
Maybe it should be `after_success` ?

Ref: https://docs.travis-ci.com/user/customizing-the-build/#the-build-lifecycle